src/components/POLine/POLineAgreementLines/POLineAgreementLinesContainer.js

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
+import {
+  Accordion,
+  Badge,
+  Loading,
+} from '@folio/stripes/components';
+import {
+  LIMIT_MAX,
+  useShowCallout,
+} from '@folio/stripes-acq-components';
+
+import {
+  AGREEMENTS,
+  AGREEMENT_LINES,
+} from '../../Utils/resources';
+
+import { ACCORDION_ID } from '../const';
+import POLineAgreementLines from './POLineAgreementLines';
+
+const POLineAgreementLinesContainer = ({ lineId, label, mutator }) => {
+  const [agreements, setAgreements] = useState();
+  const showCallout = useShowCallout();
+
+  useEffect(() => {
+    setAgreements();
+
+    mutator.agreementLines.GET()
+      .then(agreementsLinesResp => {
+        if (agreementsLinesResp.length) {
+          const agreementIds = [...new Set(agreementsLinesResp.map(({ owner }) => owner.id))];
+          const query = agreementIds.map(id => `id==${id}`).join('||');
+
+          return mutator.agreements.GET({
+            params: {
+              filters: query,
```

Comment:
`mod-agreements` module is written on Groovy and it doesn't support `params: { query }` (it returns all items with any query) and requires to use `||` instead of `or` as usual we use.